samples/Templates/Scenarios/WeatherLarge.template.json

```diff
@@ -29,26 +29,30 @@
                             "text": "{{DATE(${formatEpoch(data[0].sunrise_ts, 'yyyy-MM-ddTHH:mm:ssZ')}, SHORT)}}",
                             "weight": "Bolder",
                             "size": "Large",
-                            "wrap": true
+                            "wrap": true,
+                            "color": "Dark"
```

Comment:
A couple of things to note for `color: "Dark"`:

- `viva-connections-dark` hostConfig seems to have the dark and light colors switched.
  containerStyles.default.foregroundColors.dark => light
  containerStyles.default.foregroundColors.light => dark
  hostConfig reference: https://github.com/microsoft/AdaptiveCards/blob/cfc690077f7afc2e33f2f6caeb5f69dc3f401ae9/samples/HostConfig/viva-connections-dark.json#L53-L68
  If these should be switched, I can go ahead and make this change.

- `cortana-skills-classic` hostConfig does not set a foregroundColor dark enough to meet contrast requirements. I believe this should be updated, but I am not sure what design requirements we are trying to meet here.

Reply:
feel free to update these 👍

Reply:
Just updated :)